packages/new-polymath-ui/src/components/inputs/TimePickerSelect/TimePickerSelect.tsx

```diff
@@ -1,15 +1,24 @@
 import { range } from 'lodash';
-import React, { Component } from 'react';
+import React, { Component, FC } from 'react';
 import moment from 'moment';
 
 import { SelectPrimitive, SelectProps } from '../Select';
-import { formikProxy } from '~/components/inputs/formikProxy';
+import {
+  FormikProxy,
+  FormikExternalProps,
+} from '~/components/inputs/FormikProxy';
 
 export interface TimePickerSelectProps extends SelectProps {
   format: string;
   onChange: (value: number) => void;
   onBlur: () => void;
   value: any;
```

Comment:
Shouldn't this be typed as `number`?

Reply:
No idea, not my typing, I'll try changing it but I'm worried this cascades a set of type issues, we'll see